classes/PodsUI.php

```diff
@@ -3686,7 +3828,20 @@ public function filters_popup() {
 								</label>
 
 								<span class="pods-ui-posts-filter<?php esc_attr_e( empty( $value ) ? ' pods-hidden' : '' ); ?>">
-								<?php echo PodsForm::field( 'filter_' . $filter, $value, 'text', $options ); ?>
+								<?php
+								// Prevent p div issues.
+								echo str_replace(
+									array(
+										'<div',
+										'</div>',
+									),
+									array(
+										'<span',
+										'</span>',
+									),
+									PodsForm::field( 'filter_' . $filter, $value, 'text', $options )
+								);
```

Comment:
:warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: 32b6ecd10d2efa23a6eda7a4da114855</sub>